vhdbuilder/packer/components.json

```diff
@@ -224,12 +209,10 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.26.19",
-        "v1.26.22",
-        "v1.27.13",
-        "v1.27.16",
+        "v1.27.7",
```

Comment:
pls remove old patches, and keep newer versions
```
  {{- else if eq .component "cloud-provider-node-manager-linux" -}}
    {{- if semverCompare ">=1.29.0" .version -}}v1.29.3
    {{- else if semverCompare ">=1.28.0" .version -}}v1.28.8
    {{- else if semverCompare ">=1.27.0" .version -}}v1.27.16
    {{- else if semverCompare ">=1.26.0" .version -}}v1.26.22
```

Reply:
Have removed old patch versions and updated the patch versions to include versions in that logic.